bin/rails

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
+if ENV['RAILS_ENV'] == 'test'
+  require 'simplecov'
+  SimpleCov.start 'rails'
+  puts "required simplecov"
```

Comment:
unneeded comment?

Reply:
did not realise this from 
see https://github.com/colszowka/simplecov#getting-started